lib/std/heap/sbrk_allocator.zig

```diff
@@ -15,7 +15,7 @@ pub fn SbrkAllocator(comptime sbrk: *const fn (n: usize) usize) type {
 
         pub const Error = Allocator.Error;
 
-        lock: std.Thread.Mutex = .{},
+        //lock: std.Thread.Mutex = .{},
```

Comment:
Okay -- there was a related TODO below to start using this instead of the global, so thought I'd keep it for now. I can open an issue to track that TODO if you want.